applications/crossbar/priv/couchdb/schemas/call_recordings.json

```diff
@@ -121,11 +191,15 @@
             "name": "to",
             "type": "string"
         },
+        "ui_metadata": {
```

Comment:
these fields are not part of the schema. any field added by the client that isn't in the schema is stored as-is. MonsterUI adds these fields for its use but have nothing to do with the backend. please remove

Reply:
The schema currently is "additionalProperties": false so we can't add arbitarry fields as you suggest. I can change it to  "additionalProperties": true if you prefer

Reply:
@alanrevans it was set because these docs are considered read-only (hence why only GET/DELETE are allowed methods). Still not entirely sure on the use case being handled by these changed?